reproject/tests/test_high_level.py

```diff
@@ -117,8 +117,10 @@ def test_return_footprint(self, reproject_function):
 
 
 @pytest.mark.filterwarnings("ignore::FutureWarning")
-@pytest.mark.parametrize("projection_type, dtype", itertools.product(ALL_MODES, ALL_DTYPES))
-def test_surface_brightness(projection_type, dtype):
+@pytest.mark.parametrize(
+    "projection_type, dtype, block_size", itertools.product(ALL_MODES, ALL_DTYPES, (None, [2, 2]))
+)
+def test_surface_brightness(projection_type, dtype, block_size):
```

Comment:
Is this change still needed given the other tests which already check the blocked functionality?

Reply:
nope.  I added a test that actually reproduces the error, though: the problem was that `reproject_interp` w/blocks was tested w/header but not w/WCS as output projection, but they behave differently